controllers/storagecluster/job_templates.go

```diff
@@ -85,6 +87,13 @@ In the event of errors or invalid user inputs,
 the Job will attempt to remove as many OSDs as can be processed and complete without returning an error condition.
 Users should always check for errors and success in the log of the finished OSD removal Job.`,
 			},
+			{
+				Name:        "FORCE_OSD_REMOVAL",
+				DisplayName: "FORCE_OSD_REMOVAL",
+				Required:    false,
```

Comment:
I believe we also need to specify the default value as:
```
Value: "false",
```
Without that I assume the default is just empty, which will likely cause an error in Rook. But we need to test these template changes in an openshift cluster to confirm the default behavior.